doc/developer/DeveloperDocumentation.adoc

```diff
@@ -7,14 +7,16 @@ Here is basically what you will need to set up on your development machine:
 * *Create a database and start it:*
   ** (Optional) You can change your db root account's password in the database/instance/manage-db.sh script.
   By default it's `dev_password_to_change`.
-  ** Run `./database/instance/manage-db.sh create <FOLDER_PATH_TO_PERSISTS>`
-  ** Run `mvn liquibase:updateTestingRolling -Pdev -Dliquibase.database.password=<YOUR DB ACCOUNT PASS>`
+  ** Go to the folder containing the script `manage-db.sh`:  `cd ./database/instance`, from the project root
```

Comment:
You should remove the `./` from the path to something which makes more sense like `<ARA_ROOT>/database`

Reply:
done